src/data/generateEmissionsTable.js

```diff
@@ -1,68 +1,250 @@
-const fs = require('fs');
+//const fs = require('fs');
+//const https = require('https');
+//const Web3 = require('web3');
+
+import fs from 'fs'
+import Web3 from 'web3';
+import fetch from 'node-fetch';

```

Comment:
I'm not able to run the script locally because these dependencies aren't in the package.json
I guess you need to run `npm install --save-dev web3, node-fetch` as this script is only ran locally by maintainers and not be end-users of the npm package.
 
Also, if we force node v18 for this repo, we don't need node-fetch anymore, `fetch()` is available globally :)

Reply:
Updated to require node >v18.0 and added web3 to package.json